lasagne/layers/conv.py

```diff
@@ -458,8 +468,8 @@ def get_output_for(self, input, input_shape=None, **kwargs):
                                       image_shape=input_shape,
                                       filter_shape=self.get_W_shape(),
                                       border_mode='full')
-            shift_x = (self.filter_size[0] - 1) // 2
-            shift_y = (self.filter_size[1] - 1) // 2
+            shift_x = self.filter_size[0] // 2
+            shift_y = self.filter_size[1] // 2
             conved = conved[:, :, shift_x:input.shape[2] + shift_x,
                             shift_y:input.shape[3] + shift_y]
```

Comment:
+1 for consistency! Here it'd probably be simpler to use the `crop_x:-crop_x or None, crop_y:-crop_y or None` trick then...